Microsoft.Toolkit.Uwp.UI/Extensions/Visual/VisualEx.cs

```diff
@@ -593,39 +593,49 @@ private static void OnSizeChanged(DependencyObject d, DependencyPropertyChangedE
             }
         }
 
-        private static void OnKeepCenterPointCenteredChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        private static void OnNormalizedCenterPointChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
         {
             if (d is FrameworkElement element && !DesignTimeHelpers.IsRunningInLegacyDesignerMode)
             {
-                SetupKeepCenterPointCentered(e, element);
+                SetupNormalizedCenterPoint(e, element);
             }
         }
 
-        private static void SetupKeepCenterPointCentered(DependencyPropertyChangedEventArgs e, FrameworkElement element)
+        private static void SetupNormalizedCenterPoint(DependencyPropertyChangedEventArgs e, FrameworkElement element)
         {
             element.SizeChanged -= KeepCenteredElementSizeChanged;
 
-            if (e.NewValue is bool keepCentered)
+            if (e.NewValue is string normalizedValue)
             {
-                if (keepCentered)
+                element.SizeChanged -= KeepCenteredElementSizeChanged;
+                try
                 {
-                    element.SizeChanged -= KeepCenteredElementSizeChanged;
-                    element.SizeChanged += KeepCenteredElementSizeChanged;
+                    var vectorValue = normalizedValue.ToVector3();
```

Comment:
Can ToVector3 also support "float, float" in addition too "float" and "float, float, float"? Not sure where it's used elsewhere but for this case it'd be handy as most people don't ever use the Z component.

Or perhaps just a special case just for this property?

Reply:
I thought about that but decided against it as it felt "dirty". Most properties are Vector3 and there is a possibility that could cause unforeseen issues.